spec/Dynamic-Validator.md

```diff
@@ -67,7 +67,16 @@ The election is a process that elects validators of a term according to the foll
 
 1. Pick **MAX_NUM_OF_VALIDATORS** candidates in order of having received many delegations.
 2. Select **MIN_NUM_OF_VALIDATORS** accounts; they become validators.
-3. Among the rest of them, drop the accounts having received less than MIN_CCS_RATE_TO_BE_VALIDATOR; the remains become validators.
+3. Among the rest of them, drop the accounts that received less than **DELEGATION_THRESHOLD**; the remaining accounts become validators.
+
+This process guarantees two things:
+* There are **MIN_NUM_OF_VALIDATORS** validators if the number of candidates are larger than **MIN_NUM_OF_VALIDATORS**.
```

Comment:
Isn't it MAX_NUM_OF_VALIDATORS?

Reply:
No. It isn't. **MIN_NUM_OF_VALIDATORS** is correct.
In the current constants, CodeChain guarantees 4 validators only when there are 4 valid candidates.

Reply:
I changed the statement to remove the ambiguity.